components/layout/style_ext.rs

```diff
@@ -961,6 +972,24 @@ impl ComputedValuesExt for ComputedValues {
         };
         has_percentage(box_offsets.block_start) || has_percentage(box_offsets.block_end)
     }
+
+    // <https://drafts.csswg.org/cssom-view/#overflow-directions>
+    fn overflow_direction(&self) -> OverflowDirection {
+        let writing_mode: WritingMode = self.writing_mode;
+        let inline_end_direction = writing_mode.inline_end_physical_side();
+        let block_end_direction = writing_mode.block_end_physical_side();
```

Comment:
`: WritingMode` seems unnecessary. And the variable too.

```suggestion
        let inline_end_direction = self.writing_mode.inline_end_physical_side();
        let block_end_direction = self.writing_mode.block_end_physical_side();
```

Reply:
Fixed, thanks!